composer.json

```diff
@@ -18,14 +18,12 @@
     },
     "require-dev": {
         "doctrine/common": ">=2.1",
-        "zendframework/zend-stdlib": "~2.7",
```

Comment:
this is still necessary. The testsuite relies on the stdlib error handler in one place (currently not failing because of zend-version depending on it transitively and #10 is not yet merged)